lib/backend/firestore/firestorebk.go

```diff
@@ -345,41 +360,34 @@ func (b *Backend) GetName() string {
 // Create creates item if it does not exist
 func (b *Backend) Create(ctx context.Context, item backend.Item) (*backend.Lease, error) {
 	r := newRecord(item, b.clock)
-	_, err := b.svc.Collection(b.CollectionName).Doc(b.keyToDocumentID(item.Key)).Create(ctx, r)
+	res, err := b.svc.Collection(b.CollectionName).Doc(b.keyToDocumentID(item.Key)).Create(ctx, r)
```

Comment:
style nit, here and in the other functions

```suggestion
	res, err := b.svc.Collection(b.CollectionName).
		Doc(b.keyToDocumentID(item.Key)).
		Create(ctx, r)
```